packages/backend/src/yeomanui.ts

```diff
@@ -490,9 +490,8 @@ export class YeomanUI {
     const questions: any[] = [];
 
     let selectedWorkspaceConfig;
-
     const currentPath = _.get(vscode, "workspace.workspaceFolders[0].uri.fsPath");
-    if (!currentPath || YeomanUI.PROJECTS.toLowerCase() === currentPath.toLowerCase()) {
```

Comment:
why it is not relevant anymore ?

Reply:
the requirement in the task is to open the project in multi-root as default (unless the user changed it on the settings) also in the projects folder